glossary.md

```diff
@@ -170,6 +294,20 @@ A store refers to the storage node in the TiKV cluster (an instance of `tikv-ser
 
 Top SQL helps locate SQL queries that contribute to a high load of a TiDB or TiKV node in a specified time range. For details, see [Top SQL user document](/dashboard/top-sql.md).
 
+### TPS
+
+Transactions Per Second (TPS) is the number of transactions a database processes per second, serving as a key metric for measuring database performance and throughput.
+
 ### TSO
 
 Because TiKV is a distributed storage system, it requires a global timing service, Timestamp Oracle (TSO), to assign a monotonically increasing timestamp. In TiKV, such a feature is provided by PD, and in Google [Spanner](http://static.googleusercontent.com/media/research.google.com/en//archive/spanner-osdi2012.pdf), this feature is provided by multiple atomic clocks and GPS. For details, see [TSO](/tso.md).
+
+## U
+
+### URI
+
+Uniform Resource Identifier (URI) is a standardized format for identifying a resource. For more information, see [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) on Wikipedia.
```

Comment:
Why do we want to include this definition in the glossary?

Reply:
This is because it is mentioned in our docs quite often. I would assume URI/URL could be considered common knowledge and left out. I'm fine with it either way.

```
$ git grep -cw URI | sed "s/:/\t/" | sort -k2 -r | head
ticdc/ticdc-sink-to-mysql.md	7
tidb-cloud/config-s3-and-gcs-access.md	6
tidb-cloud/changefeed-sink-to-cloud-storage.md	6
ticdc/ticdc-sink-to-kafka.md	6
dumpling-overview.md	6
br/br-pitr-manual.md	5
tiproxy/tiproxy-api.md	4
tidb-cloud/tidb-cloud-auditing.md	4
tidb-cloud/migrate-sql-shards.md	4
tidb-cloud/migrate-from-op-tidb.md	4
```

Reply:
@qiancai @benmeadowcroft should we keep it or not? It is something well known and not TiDB specific. But on the other hand, it is often mentioned in the docs.